onnxruntime/core/framework/op_kernel.cc

```diff
@@ -104,8 +104,9 @@ Fence_t OpKernelContext::OutputFence(int index) const {
 
 Status OpKernelContext::GetOrCreateOutputMLValue(int index, OrtValue*& p_value) {
   auto output_arg_index = GetOutputArgIndex(index);
-  ORT_ENFORCE(execution_frame_->GetOrCreateNodeOutputMLValue(output_arg_index, nullptr, p_value).IsOK());
-  return Status::OK();
+  auto status = execution_frame_->GetOrCreateNodeOutputMLValue(output_arg_index, nullptr, p_value);
+  ORT_ENFORCE(status.IsOK(), status.ErrorMessage());
```

Comment:
in this case, as the function already returns a status, can we just return it?

Reply:
We could, but this is a private method and the only place that calls it also calls ORT_ENFORCE on the status. As there's no point delaying the ORT_ENFORCE call, I'll update this method to return OrtValue* instead.